nengo/networks/workingmemory.py

```diff
@@ -4,26 +4,25 @@
 from nengo.networks import EnsembleArray
 
 
-def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+def InputGatedMemory(n_neurons, dimensions, feedback=1.0,
+                     difference_gain=1.0, recurrent_synapse=0.1,
+                     difference_synapse=None, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    if difference_synapse is None:
+        difference_synapse = recurrent_synapse
 
     n_total_neurons = n_neurons * dimensions
 
     with net:
         # integrator to store value
-        with mem_config:
-            net.mem = EnsembleArray(n_neurons, dimensions,
-                                    neuron_nodes=True, label="mem")
-            nengo.Connection(net.mem.output, net.mem.input,
-                             transform=fdbk_scale)
+        net.mem = EnsembleArray(n_neurons, dimensions,
+                                neuron_nodes=True, label="mem")
+        nengo.Connection(net.mem.output, net.mem.input,
+                         transform=feedback,
+                         synapse=nengo.Lowpass(recurrent_synapse))
```

Comment:
Okay, I get it now. From now on, whenever the synapse is parameter, you don't cast it as `nengo.Lowpass`.